.php_cs

```diff
@@ -18,5 +18,6 @@ return PhpCsFixer\Config::create()
         'array_syntax' => [
             'syntax' => 'short'
         ],
+        'no_unused_imports' => false,
```

Comment:
Why do we need this? This can potentially sneak in a lot of garbage under the hood.

Reply:
New version of PHPCS uses true as a default, so I think I added this to be BC.

Reply:
Think we could set this to true, but that should be a secondary PR as it's going to cause a bunch of unrelated line drops.

Reply:
I added this when I was trying to upgrade PHPCS, which I aborted because it wasn't upgradable at the time. Should be able to upgrade now they say, but shouldn't be required for this PR imo.